models/anymal_description/director_config.json

```diff
@@ -119,5 +119,5 @@
   "handCombinations": [],
 
   "userConfig" : [],
-  "disableComponents" : ["useHumanoidDRCDemos","useConvexHullModel", "useAtlasDriver", "useFootsteps", "useControllerRate", "useCOPMonitor", "useHands", "useLCMGL"]
+  "disableComponents" : ["useHumanoidDRCDemos","useConvexHullModel", "useAtlasDriver", "useFootsteps", "useControllerRate", "useCOPMonitor", "useHands", "useLCMGL","usePlanning", "useIk"]
```

Comment:
FYI: this is not absolutely necessary as the drake planning does work but we are currently using it.